### PR TITLE
chore: bump peer dependency

### PR DESCRIPTION
**Pull request by juliuscc**
fixes: #37

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,7 +66,7 @@
     "semantic-release": "^15.13.3"
   },
   "peerDependencies": {
-    "semantic-release": ">=11.0.0 <16.0.0"
+    "semantic-release": ">=11.0.0 <17.0.0"
```
Comment (BeyondEvil) [inline package.json]: Probably less than 18 here @juliuscc Or `<=`. The latest Semantic Release is 17.0.4

Comment (juliuscc) [reply package.json]: Ah that is right!

Comment (juliuscc) [reply package.json]: That is fixed now

Comment (BeyondEvil) [reply package.json]: Will you update the beta release as well? @juliuscc

**Comment by BeyondEvil** [comment]
Sorry @juliuscc but will this `<=17.0.0` work with 17.0.4? 🙈

**Comment by juliuscc** [comment]
Good catch! Probably not.

File: package.json (hunk 1 above)
Comment (juliuscc) [reply package.json]: Yes I will do that so you can try before I merge. 2 min

**Comment by juliuscc** [comment]
Check `1.6.1-beta.1` @BeyondEvil 

```
npm i semantic-release-slack-bot@1.6.1-beta.1
```

**Comment by BeyondEvil** [comment]
Runs good @juliuscc ! Ship it! 😄 

One note tho: This PR is marked as "chore", that won't trigger a new release will it? 🤔

**Comment by juliuscc** [comment]
Another good catch! Will fix. Thanks as usual @BeyondEvil! 🌟

**Pull request merged**